modules/fedimint-wallet-server/src/lib.rs

```diff
@@ -398,6 +406,8 @@ impl ServerModule for Wallet {
 
         items.push(WalletConsensusItem::Feerate(fee_rate_proposal));
 
+        items.push(WalletConsensusItem::PegInMinimum(Amount::from_sats(10_000)));
```

Comment:
Shouldn't this detect if the current vote is already equal to this value and submit citem only if they are not equal?

Reply:
This would be a unnecessary optimisation as redundant votes are filtered out after the ordering. This would need to be done though if we ever are able to stop aleph bft when no new items need to be ordered.